kernel/src/predicates/tests.rs

```diff
@@ -117,7 +117,7 @@ fn test_default_partial_cmp_scalars() {
     }
 
     let expect_if_comparable_type = |s: &_, expect| match s {
-        Null(_) | Decimal(..) | Struct(_) | Array(_) => None,
```

Comment:
@scovich - this disabled some tests that are using partial_cmp for decimals, that are now enabled.